### ISSUE TITLE
Wrong permission on files in EGG-INFO

### ISSUE THREAD

**Issue by hanmanhui** [google/python-fire#107]
After installing fire with pip, python-mode plugin on vim started to crash with error logs like [this issue](https://github.com/python-mode/python-mode/issues/750).
Unlike the issue above, last error was like this.
`IOError: [Errno 13] Permission denied: '/usr/lib64/python2.7/site-packages/fire-0.1.1-py2.7.egg/EGG-INFO/top_level.txt'`

Files in EGG-INFO showed permission like this.
```
> ll /usr/lib64/python2.7/site-packages/fire-0.1.1-py2.7.egg/EGG-INFO/
total 28
-rw-r----- 1 root root    1 Aug 24 13:21 dependency_links.txt
-rw-r--r-- 1 root root    1 Aug 24 13:21 not-zip-safe
-rw-r--r-- 1 root root   47 Aug 24 13:21 pbr.json
-rw-r----- 1 root root 1417 Aug 24 13:21 PKG-INFO
-rw-r----- 1 root root   16 Aug 24 13:21 requires.txt
-rw-r----- 1 root root  699 Aug 24 13:21 SOURCES.txt
-rw-r----- 1 root root    5 Aug 24 13:21 top_level.txt
```

I think permissions on these files should be set to `-rw-r--r-- (644)`.

**Comment by dbieber** [comment]
Yes, I will make sure files are world readable in the upcoming release.

**Comment by dbieber** [comment]
This should be resolved starting with 0.1.3, released today.
